lib/oeqa/selftest/cases/testutils.py

```diff
@@ -124,12 +124,18 @@ def verifyProvisioned(testInst, machine):
     ran_ok = False
     for delay in [5, 5, 5, 5, 10, 10, 10, 10]:
         stdout, stderr, retcode = testInst.qemu_command('aktualizr-info')
-        if retcode == 0 and stderr == b'' and stdout.decode().find('Fetched metadata: yes') >= 0:
+        if retcode == 0 and stderr == b'':
             ran_ok = True
             break
         sleep(delay)
     testInst.assertTrue(ran_ok, 'aktualizr-info failed: ' + stderr.decode() + stdout.decode())
 
+    ran_ok = False
+    if stdout.decode().find('Fetched metadata: yes') == 0:
```

Comment:
This should be `< 0` to check for metadata not yet found.

Reply:
Ah, thanks!